app/Http/Controllers/PeopleController.php

```diff
@@ -205,6 +205,12 @@ public function update(Request $request, $contactId)
             $contact->twitter_profile_url = null;
         }
 
+        if ($request->input('linkedin') != '') {
```

Comment:
Will defer to @djaiss then. I hadn't taken the time to look at the other fields, sorry about that.

My opinion is yes trim all in another commit.